poglink/models/rates.py

```diff
@@ -132,10 +134,12 @@ def from_statuses(cls, old, new):
     # highlights changed rates in embed
     def to_embed(self):
 
-        rates_dict = self.old.to_dict()
+        rates_dict = self.old.to_dict(include_extras=False)
 
         # bold updated rates
-        updated_rates = {item.key: f"**{item.new_val}**" for item in self.items}
+        updated_rates = {
+            item.key: f"**{item.new_val}**" for item in self.items if not item.is_extra
+        }
```

Comment:
The problem originally was that this `rates_dict` already included the extra fields if there were any.

I've added the `include_extras` argument to the `to_dict` method so it can be dumped to dict without any unexpected values included.